src/main/java/Main.java

```diff
@@ -1,5 +1,7 @@
+import com.epam.tat.module4.Calculator;
+
 public class Main {
     public static void main(String[] args) {
-
+        Calculator calculator = new Calculator();
     }
 }
```

Comment:
Really? For what?

Reply:
I was checking the visibility of the class after installing the jar. I will delete it as unnecessary.